### PR TITLE
NMS-16367 : Grafana reports endpoint failure.

### PR DESCRIPTION
**Pull request by MusaidAli**
### External References

* Jira (Issue Tracker): https://opennms.atlassian.net/browse/NMS-16367

### CODE REVIEW

**Review by cgorantla — COMMENTED** [review]
We have to add a test case that prevents this from breaking again.

File: container/features/src/main/resources/features.xml [hunk 1]
```diff
@@ -1620,7 +1620,7 @@
         <!--<bundle>mvn:org.opennms.features.endpoints.grafana/org.opennms.features.endpoints.grafana.api/${project.version}</bundle>-->
         <!--<bundle>mvn:org.opennms.features.endpoints.grafana.persistence/org.opennms.features.endpoints.grafana.persistence.api/${project.version}</bundle>-->
         <!--<bundle>mvn:org.opennms.features.endpoints.grafana.persistence/org.opennms.features.endpoints.grafana.persistence.impl/${project.version}</bundle>-->
-        <bundle dependency="true">mvn:org.jetbrains.kotlin/kotlin-osgi-bundle/${kotlinVersion}</bundle>
+      <!--  <bundle dependency="true">mvn:org.jetbrains.kotlin/kotlin-osgi-bundle/${kotlinVersion}</bundle>-->
```
[the remark this answers is not in the document]
Comment (MusaidAli) [reply container/features/src/main/resources/features.xml]: Yes,reverted it .resolved. branch is tested grafana endpoint working fine.

**Review by cgorantla — COMMENTED** [review]
Validated and works fine.

@MusaidAli Please re-targe this to `foundation-2024`

**Comment by cgorantla** [comment]
Also created https://opennms.atlassian.net/browse/NMS-16944 to track tests

**Comment by cgorantla** [comment]
Override by https://github.com/OpenNMS/opennms/pull/7516

**Pull request closed without merging**